mmv1/templates/terraform/examples/workbench_instance_full.tf.erb

```diff
@@ -10,16 +10,6 @@ resource "google_compute_subnetwork" "my_subnetwork" {
   ip_cidr_range = "10.0.1.0/24"
 }
 
-resource "google_kms_key_ring" "keyring" {
-  name = "<%= ctx[:vars]['keyring_name'] %>"
-  location = "global"
```

Comment:
I realised that I've replaced a global KMS key with a us-central1 regional one. That region matches the zone the instance is in, but to be safe I'm going to run the acceptance test in TeamCity

Reply:
The test passed here, so happy to merge: https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_Google_SarahManualTestTeamcityRefactor_GOOGLE_MMUPSTREAMTESTS_GOOGLE_PACKAGE_WORKBENCH/97561?buildTab=overview&expandBuildDeploymentsSection=false&hideTestsFromDependencies=false&hideProblemsFromDependencies=false&expandBuildChangesSection=true